app/src/androidTest/java/com/droidhats/campuscompass/views/MapFragmentTest.kt

```diff
@@ -74,4 +77,34 @@ class MapFragmentTest {
         //Ensuring the text of the toggle just clicked is indeed LOY
         onView(withId(R.id.toggleButton)).check(matches(withText("LOY")))
     }
+
+    @Test
+    fun test_NavigationDrawer(){
+            //Opening side Menu
+            onView(withId(R.id.drawer_layout)).perform(DrawerActions.open())
+            onView(withId(R.id.drawer_layout)).check(matches(isOpen()));
+            Thread.sleep(2000);
+
+
+            //Checking if logo is displayed
+            onView(withId(R.id.nav_logo)).check(matches(isDisplayed()))
+
+            //Checking if title is displayed
+            onView(withId(R.id.nav_title)).check(matches(isDisplayed()))
+
+            //Checking if My places menu item is displayed
+            onView(withText("My Places")).check(matches(isDisplayed()))
+
+            //Checking if Schedule menu item is displayed
+            onView(withText("Schedule")).check(matches(isDisplayed()))
+
+            //Checking if Resources menu item is displayed
+            onView(withText("Resources")).check(matches(isDisplayed()))
+
+            //Checking if Explore menu item is displayed
+            onView(withText("Explore")).check(matches(isDisplayed()))
+
+            //Checking if Settings menu item is displayed
+            onView(withText("Settings")).check(matches(isDisplayed()))
+   }
```

Comment:
Great work! I ran the test and it works perfectly fine. 
However, the side menu test should be on its own file and not under mapFragmentTest.

Please move this test to a new file 😄